coalib/bearlib/languages/documentation/AssembledDocumentation.py

```diff
@@ -0,0 +1,71 @@
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+
+Description = DocumentationComment.Description
+Parameter = DocumentationComment.Parameter
+ReturnValue = DocumentationComment.ReturnValue
+
+
+def assemble_documentation(doccomment, param_sym, ret_sym):
+    """
+    Assembles a list of parsed documentation comments.
+
+    This function just assembles the documentation comment
+    itself, without the markers and indentation.
+
```

Comment:
Does it really need one though? I mean this isn't one independent class, it depends on a lot of other classes, plus I'd have to set up a stuff. Even if I write one, it won't be succinct for sure.